Mail/Views/Thread/Message/MessageBodyView.swift

```diff
@@ -45,23 +45,20 @@ struct MessageBodyView: View {
             VStack {
                 if presentableBody.body != nil {
                     WebView(model: model, messageUid: messageUid) {
-                        loadBody(blockRemoteContent: blockRemoteContent)
+                        loadBody(presentableBody: presentableBody, blockRemoteContent: blockRemoteContent)
                     }
                     .frame(height: model.webViewHeight)
                     .onAppear {
-                        loadBody(blockRemoteContent: blockRemoteContent)
-                    }
-                    .onChange(of: presentableBody) { _ in
-                        loadBody(blockRemoteContent: blockRemoteContent)
+                        loadBody(presentableBody: presentableBody, blockRemoteContent: blockRemoteContent)
                     }
-                    .onChange(of: isMessagePreprocessed) { _ in
-                        loadBody(blockRemoteContent: blockRemoteContent)
+                    .onChange(of: presentableBody) { newPresentableBody in
+                        loadBody(presentableBody: newPresentableBody, blockRemoteContent: blockRemoteContent)
```

Comment:
Here using `newPresentableBody` fixed an issue where the processing was performed on the _previous_ value of presentableBody.